test/02-investigate.t

```diff
@@ -3,10 +3,7 @@
 source test/init
 bpan:source bashplus +err +fs +sym
 
-plan tests 88
-
-host=localhost
```

Comment:
This is not necessary anymore, and you rather want test `investigation_gid=...`

Reply:
Do u mean to revert the host?

Reply:
yes, and the whole change. why would this be needed?
Instead test for setting `investigation_gid=...`

Reply:
I put back the host+url. However I dont know where the url used. seems unused. and I am not sure why host is needed